examples/window-app/common/src/WindowApp.cpp

```diff
@@ -113,7 +117,11 @@ WindowApp::Cover * WindowApp::GetCover(chip::EndpointId endpoint)
 CHIP_ERROR WindowApp::Init()
 {
     // Initialize device attestation config
+#ifdef EFR32_ATTESTATION_CREDENTIALS
+    SetDeviceAttestationCredentialsProvider(EFR32::GetDACProvider());
+#else
     SetDeviceAttestationCredentialsProvider(Examples::GetExampleDACProvider());
+#endif
```

Comment:
This should be done in `examples/window-app/efr32/src/main.cpp`, around where `chip::DeviceLayer::SetDeviceInfoProvider` is done

Reply:
Moved to main.cpp